django-rgd/client/rgd_client/plugin.py

```diff
@@ -91,8 +91,22 @@ def create_file_from_url(
         # Verify that url is valid in shape, will raise error on failure
         validators.url(url)
 
+        # Check if url/collection combination already exists, and return it
+        payload = {'url': url}
+        if collection is not None:
+            payload['collection'] = collection
+        data = self.session.get('checksum_file', params=payload).json()
+        # TODO: This is absolutely stumping me...
+        if isinstance(data, list) and data:
+            # Test env returns list
+            return data[0]
+        elif isinstance(data, dict) and data['results']:
+            # User env returns dict
+            return data['results'][0]
```

Comment:
I cannot figure this out... the testing environment returns a list in that get response whereas my user env returns a dict. There might be a configuration difference between the testing project and the `example_project`.

Look into later... need to merge.

Reply:
It's definitely a DRF configuration thing in the test project because this worked in https://github.com/ResonantGeoData/RD-WATCH/pull/42